app/src/main/java/com/github/dedis/student20_pop/model/PollEvent.java

```diff
@@ -0,0 +1,56 @@
+package com.github.dedis.student20_pop.model;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Class modelling a poll event
+ */
+public final class PollEvent extends Event {
+    private final Date startDate;
+    private final Date endDate;
+    private final Date startTime;
+    private final Date endTime;
+    private final List<String> choices;
+    private final boolean oneOfN;
```

Comment:
Is this attribute for single choice poll ?

Reply:
Yes, it's either "choose one of N" (default) or "approve any of N" (where N is the number of choices).